opengrok-web/src/main/java/org/opengrok/web/api/v1/controller/ProjectsController.java

```diff
@@ -189,6 +211,7 @@ public void deleteProject(@PathParam("project") final String projectName)
                     }
                 }).collect(Collectors.toSet()));
```

Comment:
could you use filter here to ensure that "" wont go further? it's just asking for troubles in future

Reply:
Good idea.